quickstart/101-virtual-network-manager-create-mesh/main.tf

```diff
@@ -0,0 +1,142 @@
+
+# Create the Resource Group
+
+resource "random_pet" "rg_name" {
+  prefix = var.resource_group_name_prefix
+}
+
+resource "azurerm_resource_group" "rg" {
+  location = var.resource_group_location
+  name     = random_pet.rg_name.id
+}
+
+# Create three virtual networks
+
+resource "random_pet" "virtual_network_name" {
+  prefix = "vnet"
+}
+resource "azurerm_virtual_network" "vnet_01" {
+  name                = "${random_pet.virtual_network_name.id}-01"
+  resource_group_name = azurerm_resource_group.rg.name
+  location            = azurerm_resource_group.rg.location
+  address_space       = ["10.0.0.0/16"]
+
+  depends_on          = [azurerm_resource_group.rg]
```

Comment:
Since `azurerm_virtual_network.vnet_01` used `azurerm_resource_group.rg.name` explicitly, I think we can remove this `depends_on`.

Reply:
updated.